### PR TITLE
Update contributors list to reflect current status

### PR DESCRIPTION
**Pull request by btashton**
Signed-off-by: Brennan Ashton <bashton@brennanashton.com>

### CODE REVIEW

File: _data/contributors.yml [hunk 1]
```diff
@@ -66,6 +66,11 @@
   role: PPMC, Committer
   org:
 
+- name: Haitao Liu
```
Comment (xiaoxiang781216) [inline _data/contributors.yml:69]: Should we remove the extra space before "Flavio Paiva Junqueira" at line 57?

Comment (btashton) [reply _data/contributors.yml:69]: There is exactly one space between all of the entries. I think it is correct?

Comment (xiaoxiang781216) [reply _data/contributors.yml:69]: Yes, line 69 is right, but line 57 is wrong.

Comment (btashton) [reply _data/contributors.yml:69]: Sorry I'm still not seeing what you are talking about, can you use the suggest an edit?

Comment (xiaoxiang781216) [reply _data/contributors.yml:69]: Sorry, it seem that my browser render the layout a little bit wrong.

**Comment by xiaoxiang781216** [comment]
@btashton the pushlish fail:
remote: Permission to apache/incubator-nuttx-website.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/apache/incubator-nuttx-website/': The requested URL returned error: 403

**Comment by btashton** [comment]
Thanks I'll fix it tonight. The publish needs to be triggered by the push to master or it won't have the correct permissions.

https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token

**Comment by btashton** [comment]
@xiaoxiang781216 this should be resolved with #21

**Pull request merged**